templates/components/base/global_nav.html.twig

```diff
@@ -1,767 +1,62 @@
-                                        <path class="chevron-left"
-                                              d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"/>
-                                    </svg>
-                                    Contact</a>
-                            </li>
-                            <li>
-                                <a class="with-icon--before with-icon--larger" href="../page.html">
-                                    <svg class="icon icon--larger" xmlns="http://www.w3.org/2000/svg"
-                                         focusable="false" aria-hidden="true" viewBox="0 0 320 512"
-                                         width="1em" height="1em">
-                                        <path class="chevron-right"
-                                              d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"/>
-                                        <path class="chevron-left"
-                                              d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"/>
-                                    </svg>
-                                    Policies and legal information</a>
-                            </li>
-                        </ul>
+                        {% if navItem.children is defined and navItem.children|length > 0 %}
+                            <ul class="clean-list" role="list">
+                                {% for child in navItem.children %}
+                                    <li>
+                                        <a class="with-icon--before with-icon--larger" href="{{ child.url }}">
+                                            <svg xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 320 512"
+                                                 class="icon icon--larger" focusable="false" aria-hidden="true" width="1em"
+                                                 height="1em">
+                                                <use class="chevron-left"
+                                                     href="{{ asset('svg/nav-icons.svg#chevron-left', 'main') }}"></use>
+                                                <use class="chevron-right"
+                                                     href="{{ asset('svg/nav-icons.svg#chevron-right', 'main') }}"></use>
+                                            </svg>
```

Comment:
This SVG doesn't display correctly for me now (the markup looks a little different to the previous inline SVG that was being used).